### DISCUSSION TITLE
Is there a way i can get homebrew to use the already existing version of CLT?

### DISCUSSION THREAD

**Discussion by Ramos159** [Homebrew/discussions#3862] [Getting started]
Today I had installed version 13.4.1 of XCode, but i noticed homebrew used 14.1 of CLT. is there a way to get homebrew to use the version i have installed with XCode? I feel like this might cause some headaches later during development

**Comment by SMillerDev** [comment]
Homebrew packages are only compatible with the latest CLT. Not using that will cause you a collection of headaches on its own.

**Comment by Bo98** [comment]
If you are using macOS 13, you must have 14.1 as previous versions don't contain the macOS 13 SDK.

If you are using macOS 12, you can use either version (anything 13.1 or later). We only test the latest but nothing will stop you using an older version.
